packages/extension/cypress/e2e/trustlines.cy.ts

```diff
@@ -84,7 +84,6 @@ describe('Trustline', () => {
     cy.contains('Issuer:').next().should('have.text', DESTINATION_ADDRESS);
     cy.contains('Currency:').next().should('have.text', CURRENCY);
     cy.contains('Limit:').next().should('have.text', `10,000,000 ${CURRENCY}`);
-    cy.contains('Fees:').next().should('have.text', '0 XRP');
```

Comment:
Because this field makes no sense, I chose to use the same field for the estimated fees or the manual fees. in case manual fees are provided, an annotation (Manual) is added.